Первое задание диплома

Vrazhnov opened the pull request.

Commit:
Первое задание диплома

nikolay-977 reviewed (commented):
Можно улучшить: в заготовке проекта есть лишние файлы, такие как test/java/tests/read.me. Их следует удалять.

nikolay-977 commented on src/test/java/praktikum/IngredientTypeTest.java:
```diff
@@ -0,0 +1,34 @@
+        this.price = price;
+    }
+
+    @Parameterized.Parameters
```
Можно улучшить: В параметризованных тестах для аннотации `@Parameterized.Parameters` лучше использовать аргумент name: `@Parameterized.Parameters(name = "Стоимость булочки. Тестовые данные: {0} {1}")`, где {0}, {1} - индексы параметров. С аргументом name наименование теста отобразится в отчете. А использование индексов параметров повысит информативность проверки.

Commit:
Исправления замечаний:
1) удалены лишник файлы
2) выполнена оптимизация (cleanup code, optimize imports и rearrange entries)
3) В параметризованный тест добавлена аннотация с именем теста и индексами
